platform-validation/scripts/compatibility.sh

```diff
@@ -532,12 +532,82 @@ check_docker_api_version(){
 
 }
 
+check_shared_library_dependency(){
+    wrap_debug "Checking shared library dependency for aziot-edged and aziot-identityd"
+
+    # set the crucial libaries in the variable
+    # IOTEDGE_COMMON_SHARED_LIBRARIES is for both aziot-edged and aziot-identityd
+    IOTEDGE_COMMON_LIBRARIES="libssl.so.1.1 libcrypto.so.1.1 libdl.so.2 librt.so.1 libpthread.so.0 libc.so.6 libm.so.6 libgcc_s.so.1"
```

Comment:
Can you add a Todo here that we need to make sure that for every release, these binaries stay consistent?

Reply:
ok will add in next PR.